README.md

```diff
@@ -14,18 +14,95 @@
 
 </div>
 
-Vello is a 2d graphics rendering engine, using [`wgpu`].
+Vello is an experimental 2d graphics rendering engine written in Rust, using [`wgpu`].
 It efficiently draws large 2d scenes with interactive or near-interactive performance.
 
-<!-- Impressive picture here -->
-
-It is used as the rendering backend for [Xilem], a UI toolkit.
-
 Quickstart to run an example program:
 ```shell
 cargo run -p with_winit
 ```
 
+![image](https://github.com/linebender/vello/assets/8573618/cc2b742e-2135-4b70-8051-c49aeddb5d19)
+
+It is used as the rendering backend for [Xilem], a native Rust GUI toolkit.
+
+
+## Motivation
+
+Vello is meant to fill the same place in the graphics stack as other vector graphics renderers like [Skia](https://skia.org/), [Cairo](https://www.cairographics.org/), and its predecessor project [Piet](https://www.cairographics.org/).
```

Comment:
Joining in, the link for Piet should be a link to Piet. (Linking to Piet-GPU is harder since the repo was renamed to this one.)

Reply:
@waywardmonkeys this was fixed, please retract request for change or just approve, so that we can get the PR unblocked.

There's still a few other changes to be made, but wouldn't want to get a surprise block eventually on this stale issue. 👍